app/models/ncr/work_order.rb

```diff
@@ -104,7 +104,11 @@ def setup_approvals_and_observers(approving_official_email)
         self.proposal.update(status: 'approved')
       else
         approvers = emails.map{|e| User.for_email(e)}
+        original_approver_groups = self.proposal.approvers.each.map{|approver| 
+          {user: approver, active: self.proposal.is_active_approver?(approver)}
+        }
         self.proposal.approvers = approvers
+        email_removed_approvers(original_approver_groups)
```

Comment:
I think this could be simplified to

```ruby
removed_approvers = self.proposal.approvers - approvers
Dispatcher.on_approver_removal(self.proposal, removed_approvers)
```